docs/contributing.rst

```diff
@@ -96,6 +96,26 @@ systems, ``evmone.dll`` on Windows systems and ``libevmone.dylib`` on macOS.
 On macOS some of the testing scripts expect GNU coreutils to be installed.
 This can be easiest accomplished using Homebrew: ``brew install coreutils``.
 
+Search paths
+------------
```

Comment:
Shouldn't this be one level lower?

Reply:
Maybe it's best to just extend the first paragraph a little that already explains where the library is looked for?

Reply:
I firsth thought about exactly that. will change it.